Client/Frontend/Settings/CustomSearchViewController.swift

```diff
@@ -255,16 +258,20 @@ class CustomSearchEngineTextView: Setting, UITextViewDelegate {
         textField.delegate = self
         textField.backgroundColor = theme.colors.layer5
         textField.textColor = theme.colors.textPrimary
+        textField.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
should i use `build()` here also?

Reply:
Please see [55bc9ca](https://github.com/mozilla-mobile/firefox-ios/pull/17089/commits/55bc9ca535b93eae82cd124f648c27c799309f00)